core/transactor/tracker/noncer.go

```diff
@@ -72,9 +73,11 @@ func (n *Noncer) refreshNonces(ctx context.Context) {
 		// TODO: handle case where stored & chain pending nonce is out of sync?
 	}
 
-	if content, err := n.ethClient.TxPoolContent(ctx); err == nil {
-		for _, tx := range content["queued"][n.sender.Hex()] {
-			n.queuedNonces[tx.Nonce()] = struct{}{}
+	// Use txpool.inspect instead of txpool.content. Less data to fetch.
+	if content, err := n.ethClient.TxPoolInspect(ctx); err == nil {
+		for nonceStr := range content["pending"][n.sender] {
```

Comment:
dont change this to "pending" without changing queuedNonces to pendingNonces and all associated vars/comments all throughout

Reply:
good catch